src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageBlobTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.ServiceManagemenet.Common.Models;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.Management.Storage.Test.ScenarioTests
+{
+    public class StorageBlobTests : RMTestBase
+    {
+        public StorageBlobTests(ITestOutputHelper output)
+        {
+            XunitTracingInterceptor.AddToContext(new XunitTracingInterceptor(output));
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestStorageBlobContainer()
+        {
+            TestController.NewInstance.RunPsTest("Test-StorageBlobContainer");
```

Comment:
If you pull in latest from preview you will see that a new parameter has been added - this should be
```
TestController.NewInstance.RunPsTest("Test-StorageBlobContainer", interceptor);
```

And you will need to create a local XunitTracingInterceptor and add that at line 28 (see other test cs files for example).